src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -107,6 +111,64 @@ public void removePerson(Person key) {
         persons.remove(key);
     }
 
+    /// student-level operations
+
+    /**
+     * Returns true if a student with the same identity as {@code student} exists in the address book.
+     */
+    public boolean hasStudent(Student student) {
+        requireNonNull(student);
+        return students.contains(student);
+    }
+
+    /**
+     * Adds a student to the address book.
+     * The student must not already exist in the address book.
+     */
+    public void addStudent(Student s) {
+        students.add(s);
+    }
+
+    /**
+     * Replaces the given student {@code target} in the list with {@code editedStudent}.
+     * {@code target} must exist in the address book.
+     * The question identity of {@code editedStudent} must not be the same as another existing student in the address
```

Comment:
Let's change 'question' identity change to 'student' identity

Reply:
👍